packages/protocol/contracts/L1/tiers/TierProviderBase.sol

```diff
@@ -48,9 +48,20 @@ abstract contract TierProviderBase is ITierProvider {
             });
         }
 
-        if (_tierId == LibTiers.TIER_SGX2) {
+        if (_tierId == LibTiers.TIER_TDX) {
```

Comment:
It seems (since `TIER_TDX == 200` as well ass `TIER_SGX` and `TIER_TEE_ANY`) that one is enough, also their fields are the same. 
Is there any reason having all defined ?

Reply:
With different names but same IDs, in TierProviderBase, we can configure all of them in the one single file so they have different verifiers.